immuclient panics in cli mode

**What happened**

```
aman@legion:~/gocode/src/github.com/codenotary/immudb|master⚡ ⇒  ./immuclient it       
immuclient>get 1
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x88 pc=0xb1fdae]

goroutine 1 [running]:
github.com/codenotary/immudb/cmd/immuclient/immuc.(*immuc).Get.func1(0x0, 0x0, 0x5, 0xc0001db908, 0x451158, 0xc000400021)
	/home/aman/gocode/src/github.com/codenotary/immudb/cmd/immuclient/immuc/getcommands.go:90 +0x2e
github.com/codenotary/immudb/cmd/immuclient/immuc.(*immuc).Execute(0xc00030cb90, 0xc0001db9e8, 0x1, 0xc000400020, 0x1, 0x8)
	/home/aman/gocode/src/github.com/codenotary/immudb/cmd/immuclient/immuc/init.go:112 +0x54
github.com/codenotary/immudb/cmd/immuclient/immuc.(*immuc).Get(0xc00030cb90, 0xc000406010, 0x1, 0x1, 0x7f2e7c0fa830, 0xc000372398, 0x0, 0xc000372380)
	/home/aman/gocode/src/github.com/codenotary/immudb/cmd/immuclient/immuc/getcommands.go:89 +0x103
github.com/codenotary/immudb/cmd/immuclient/cli.(*cli).getKey(...)
	/home/aman/gocode/src/github.com/codenotary/immudb/cmd/immuclient/cli/getcommands.go:24
github.com/codenotary/immudb/cmd/immuclient/cli.(*cli).runCommand(0xc00030cbe0, 0xc000406000, 0x2, 0x2)
	/home/aman/gocode/src/github.com/codenotary/immudb/cmd/immuclient/cli/cli.go:232 +0x3c9
github.com/codenotary/immudb/cmd/immuclient/cli.(*cli).Run(0xc00030cbe0)
	/home/aman/gocode/src/github.com/codenotary/immudb/cmd/immuclient/cli/cli.go:127 +0x32f
github.com/codenotary/immudb/cmd/immuclient/command.(*commandline).interactiveCli.func1(0xc000356280, 0x145e2b0, 0x0, 0x0, 0x0, 0x0)
	/home/aman/gocode/src/github.com/codenotary/immudb/cmd/immuclient/command/misc.go:91 +0x4a
github.com/spf13/cobra.(*Command).execute(0xc000356280, 0x145e2b0, 0x0, 0x0, 0xc000356280, 0x145e2b0)
	/home/aman/gocode/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0xc000330a00, 0x1c, 0x20, 0xc000358601)
	/home/aman/gocode/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
	/home/aman/gocode/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/codenotary/immudb/cmd/immuclient/command.Execute(0xc000330a00, 0xc0001dbf48, 0x4447f7)
	/home/aman/gocode/src/github.com/codenotary/immudb/cmd/immuclient/command/cmd.go:33 +0xb1
main.main()
```

**What you expected to happen**

return the result

**How to reproduce it (as minimally and precisely as possible)**

```shell
./immuclient it
get 1
```

**Environment**
```shell
immuclient 1.0.1
Commit  : 0254827c54f75f70ff8530e66f927b48a3dc0a37
```

--- a/cmd/immuclient/command/misc.go
+++ b/cmd/immuclient/command/misc.go
@@ -90,6 +90,9 @@ func (cl *commandline) interactiveCli(cmd *cobra.Command) {
 		Aliases: []string{"cli-mode"},
 		Example: cli.Init(cl.immucl).HelpMessage(),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := cl.immucl.Connect(args); err != nil {
+				cl.quit(err)
+			}
 			cli.Init(cl.immucl).Run()
 			return nil
 		},

